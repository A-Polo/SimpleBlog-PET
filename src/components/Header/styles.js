import theme from '../../assets/theme/theme';

export default {
  header: {
    height: 50,
    background: theme.headerBackground,
    padding: [0, 20],
    lineHeight: '50px',
  },
  title: {
    margin: 0,
    fontSize: '16px',
  },
};
