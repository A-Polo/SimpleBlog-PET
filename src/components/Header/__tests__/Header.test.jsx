import React from 'react';
import { shallow } from 'enzyme';
import Header from '../index';

describe('component/Header', () => {
  const testProps = 'ClassName';

  describe('props', () => {
    describe('.classes', () => {
      it('should have class', () => {
        const wrapper = shallow(<Header className={testProps} />);

        expect(wrapper).toHaveClassName('ClassName');
      });
    });
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
