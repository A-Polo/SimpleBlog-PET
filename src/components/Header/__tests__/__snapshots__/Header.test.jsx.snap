// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/Header matches snapshot 1`] = `
<Header
  classes={
    Object {
      "header": "Header-header-0-1-1",
      "title": "Header-title-0-1-2",
    }
  }
/>
`;
